pkg/banneduser/banneduser.go

```diff
@@ -39,15 +39,19 @@ func NewBannedUser(userSignup *toolchainv1alpha1.UserSignup, bannedBy string) (*
 }
 
 // IsAlreadyBanned checks if the user was already banned
```

Comment:
I guess it would make sense to rename the function now.
```suggestion
// GetBannedUser returns BannedUser with the provided user email hash if found. Otherwise it returns nil.
```

Reply:
Makes sense! Thanks!